packages/api-v4/src/cloudpulse/types.ts

```diff
@@ -125,7 +132,8 @@ export interface CloudPulseMetricsRequest {
   filters?: Filters[];
   aggregate_function: string;
   group_by: string;
-  relative_time_duration: TimeDuration;
+  relative_time_duration: TimeDuration | undefined;
```

Comment:
Looks like these a bit unusual compare to other types in api-v4. Can we check with API team to omit these from the request if they are not always required ?

Reply:
@cpathipa  having relative_time_duration as undefined here makes the API request not to have the  relative_time_duration parameter itself.  As of today API supports both absolute_time_duration and relative_time_duration, and from UI we should be passing only one key at a time.

Reply:
I feel like 

```suggestion
  relative_time_duration?: TimeDuration;
```

would be more appropriate